src/payment/PaymentForm.jsx

```diff
@@ -200,13 +203,19 @@ export class PaymentFormComponent extends React.Component {
         <CardDetails submitting={submitting} />
         <div className="row justify-content-end">
           <div className="col-lg-6 form-group">
-            <button type="submit" className="btn btn-primary btn-lg btn-block" disabled={submitting} onClick={this.handleSubmitButtonClick}>
-              <FormattedMessage
-                id="payment.form.submit.button.text"
-                defaultMessage="Place Order"
-                description="The label for the payment form submit button"
-              />
-            </button>
+            {
+              loading ? (
```

Comment:
Nit: this doesn't have to change, but I'm used to using `A ? B : C` for short one-liners.  For multi-line conditionals, I prefer `if/else`, but maybe I'm just old, or not up to frontend best practices?

Reply:
In JSX I think this the only way to do it. I could pull it all out to a function instead, which is probably the preferred approach when things start to get big.

Reply:
I see.  Thanks.  Yes - it could be a function.